api/src/main/java/org/openmrs/ConditionClinicalStatus.java

```diff
@@ -39,5 +39,32 @@
 	 * Remission is where the patient is no longer experiencing the symptoms of the condition, 
 	 * but there is a risk of the symptoms returning.
 	 * */
-	HISTORY_OF 
+	@Deprecated
+	HISTORY_OF,
+	
+	/**
+	 * The Patient is experiencing a re-occurence or repeating of a previously resolved condition, 
+	 * e.g. urinary tract infection, pancreatitis, cholangitis, conjunctivitis.
```

Comment:
`@since` annotation recommended for new guys. https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NewPublicMethodsandClasses